sourcemash/frontend/static/js/views/register_modal_view.js

```diff
@@ -0,0 +1,41 @@
+Sourcemash.Views.RegisterModalView = Backbone.View.extend({
+    template: JST['register-modal'],
+
+    events: {
+        'submit #register': 'registerFromModal',
+    },
+
+    registerFromModal: function(e) {
+        e.preventDefault();
+        var formData = JSON.stringify($("#register").serializeObject());
+        $('#register-password').val('');
+        $('#register-password-confirm').val('');
+        var posting = $.ajax({
+                          type: "POST",
+                          url: "/register",
+                          data: formData,
+                          success: this.showErrors,
```

Comment:
On success showErrors? will errors ever appear? Also, shouldn't the errors automatically populate because of the code we wrote in default.js?